src/database.rs

```diff
@@ -0,0 +1,15 @@
+use sqlx::{Error, postgres::{PgPool, PgConnectOptions}};
+
+use crate::constants;
+
+pub async fn create_conn_pool() -> Result<PgPool, Error> {
+    // https://www.postgresql.org/docs/current/libpq-envars.html
+    // https://docs.rs/sqlx/latest/sqlx/postgres/struct.PgConnectOptions.html
+    let options = PgConnectOptions::new();
+    let pool = match PgPool::connect_with(options).await {
+        Ok(pool) => pool,
+        Err(_) => PgPool::connect(constants::DB_URL).await?
+    };
+    sqlx::migrate!().run(&pool).await?;
+    Ok(pool)
```

Comment:
Can you sum it up and prepare runbook in readme how to start backend? Eventually, comment here where the config comes from.

Reply:
A template for `docker compose` how do I start postgres server:
1. `mkdir postgres data`
2. Copy this into `postgres/compose.yml`
    * Fill password and db name
    * Default username: `postgres`
```
# Start with
# docker-compose up -d
version: '3.9'

services:
  postgres:
    image: postgres:15.2
    restart: always
    environment:
      POSTGRES_PASSWORD: <password>
      POSTGRES_DB: <database-name>
    ports:
      - 5432:5432
    volumes:
      - ../data:/var/lib/postgresql/data
```
3. `cd postgres && docker-compose up`

Reply:
You can feel it with dev values

Reply:
I'll look into this, thanks